vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -192,6 +214,78 @@ module Tuples =
     printfn "Struct Tuple: %A\nReference tuple made from the Struct Tuple: %A" sampleStructTuple (sampleStructTuple |> convertFromStructTuple)
 
 
+/// The F# pipe operators ('|>', '<|', etc.) and F# composition operators ('>>', '<<')
+/// are used extensively when processing data.  These operators are themselves functions
+/// which make use of Partial Application.
```

Comment:
I think Partial Application need to be capitalized - if anything put it in quotes?  Either is OK as long as style in the document is consistent.